pom.xml

```diff
@@ -57,42 +48,34 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>org.eclipse.jetty</groupId>
-        <artifactId>jetty-client</artifactId>
-        <version>${jetty.version}</version>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.361.x</artifactId>
+        <version>1750.v0071fa_4c4a_e3</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+      <dependency> <!-- TODO pending https://github.com/jenkinsci/github-branch-source-plugin/pull/624 -->
```

Comment:
released in https://github.com/jenkinsci/github-branch-source-plugin/releases/tag/1701.v00cc8184df93 AFAICT